src/big/big_decimal.cr

```diff
@@ -100,6 +101,10 @@ struct BigDecimal < Number
     num
   end
 
+  def - : BigDecimal
+    self * BigDecimal.new(-1)
```

Comment:
Just create a new `BigDecimal` with `-@value`??

Reply:
I'm not sure about that, @vegai would you chime in?

Reply:
Fixed.